# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kde31/kdenetwork31/kmime_codec_qp.h.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Use of tainted arguments in exec is deprecated at /var/www/cgi-bin/cvsweb.cgi line 2043.
===================================================================
RCS file: /home2/webcvs/mirror/kdepim/libkdenetwork/kmime_codec_qp.h,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -p -r1.4 -r1.5
--- kdepim/libkdenetwork/kmime_codec_qp.h	2003/07/26 14:51:52	1.4
+++ kdepim/libkdenetwork/kmime_codec_qp.h	2003/10/30 14:15:30	1.5
@@ -54,19 +54,11 @@ public:
     int result = 3*insize;
     // then after 25 hexchars comes a soft linebreak: =(\r)\n
     result += (withCRLF ? 3 : 2) * (insize/25);
-
+    
     return result;
   }
 
-  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const {
-    // all chars unencoded:
-    int result = insize;
-    // but maybe all of them are \n and we need to make them \r\n :-o
-    if ( withCRLF )
-      result += insize;
-
-    return result;
-  }
+  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const;
 
   Encoder * makeEncoder( bool withCRLF=false ) const;
   Decoder * makeDecoder( bool withCRLF=false ) const;
@@ -92,11 +84,7 @@ public:
     return 3*insize;
   }
 
-  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const {
-    (void)withCRLF; // keep compiler happy
-    // equally simple: nothing is encoded at all, so:
-    return insize;
-  }
+  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const;
 
   Encoder * makeEncoder( bool withCRLF=false ) const;
   Decoder * makeDecoder( bool withCRLF=false ) const;
@@ -121,11 +109,7 @@ public:
     return 3*insize;
   }
 
-  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const {
-    (void)withCRLF; // keep compiler happy
-    // same as for "q" encoding:
-    return insize;
-  }
+  int maxDecodedSizeFor( int insize, bool withCRLF=false ) const;
 
   Encoder * makeEncoder( bool withCRLF=false ) const;
   Decoder * makeDecoder( bool withCRLF=false ) const;
